Zend/zend_observer.c

```diff
@@ -338,10 +332,17 @@ ZEND_API void ZEND_FASTCALL zend_observer_fiber_switch_notify(zend_fiber_context
 	zend_llist_element *element;
 	zend_observer_fiber_switch_handler callback;
 
+	if (EG(current_execute_data) == NULL) {
```

Comment:
Perhaps this should be `from->status == ZEND_FIBER_STATUS_DEAD`?

Reply:
I suppose that's effectively equivalent.

Reply:
Pretty much for the implementation of Fiber, but the fiber status is set by the context functions, whereas this pointer might not be set properly by some extension.

Reply:
Okay, done :-)